docs/operations/compactor/configuration.md

```
@@ -82,6 +82,10 @@ partition_lateness = "10m"      # only compact hours that closed 10m ago
 memory_limit_mb = 512           # rewrites spill past this instead of growing the heap
 ```
 
+> **Removed setting (breaking change, issue #925):**
+>
+> - `[compactor.orphan_cleanup] revalidate_before_delete` no longer exists. Re-validation now runs unconditionally before any real deletion: orphan detection derives its live set from the retained snapshots' manifests and is correct on its own, so re-validation is defense-in-depth rather than the switch that made cleanup safe. A dry run skips it, since it deletes nothing. The key is silently ignored if left in a config file — these structs do not reject unknown keys — so remove it when upgrading.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)